docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/front-end/src/legacy/styles/_general.scss

```diff
@@ -456,20 +456,21 @@ div.previous-next-links {
   }
 }
 
-/* fix for Chrome problem with rendering sup tags */
+/* copying sub and sup styling from USWDS for use as nci classes */
 .sup,
-sup {
-  font-size: 80%;
-  position: relative;
-  bottom: 0.6em;
-  vertical-align: text-top; //[WCMSFEQ-980] Added vertical-align:text-top to sup styling to remove white line appearing at bottom of pages in IE
-}
-.sub,
-sub {
-  font-size: 80%;
-  position: relative;
-  bottom: -0.2em;
+.sub {
+	font-size: 75%;
```

Comment:
ideally, I would remove this rule altogether if I could remove it from normalize but it also gets added for legacy with the potential to mess things up for DCEG.  I can't extend sup/sub because it sees the it would extend the 'normalized' version of sup/sub which wouldn't work because the normalize happens after uswds's.